### PR TITLE
config: add document for items which will be removed

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### What is changed, added or deleted? 

Some config items will be removed from tidb-ansible template, in case no one knows them in
the future, we should document them.

### What is the related PR or file link(s)? 

https://github.com/pingcap/tidb-ansible/pull/978

### Which version does your change affect? 

3.0 and dev

### CODE REVIEW

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 1]
```diff
@@ -1028,6 +1028,25 @@ raftdb 相关配置项。
...
 + 默认值：/tmp/tikv/store
 
+## security
+
+安全相关配置项。
+
+### `ca-path`
+
++ CA 文件路径
++ 默认：""
```
Comment (dcalvin) [inline dev/reference/configuration/tikv-server/configuration-file.md]: Please be consistent in the use of “默认” and “默认值” in the same file.

Comment (lucklove) [reply dev/reference/configuration/tikv-server/configuration-file.md]: OK

**Comment by ericsyh** [comment]
LGTM

**Comment by lilin90** [comment]
@lucklove Still inconsistent among different files. And please sign the CLA.

**Comment by lucklove** [comment]
> * Still inconsistent among different files.
I notice that pd use different word compare with tidb and tikv, but it's not caused by this PR, it should be fixed in another pull request.

**Comment by lilin90** [comment]
@dcalvin PTAL

**Comment by dcalvin** [comment]
LGTM. Agree that we may need to fix the inconsistency in another PR.

**Pull request merged**